### ISSUE TITLE
ColecoVision and Intellivision not Working properly

### ISSUE THREAD

**Issue by CLIprogrammer** [OpenEmu/OpenEmu#3270]
I have downloaded full ROM sets for both ColecoVision and Intellivision.  Both do not work for different reasons and are explained below.

1.  Select a game in ColecoVision and it opens into an empty screen

md5 hashtag -

tony-powells-mac-pro-2:~ tonypowell$ md5 /Users/tonypowell/Documents/Full_Rom_Sets/Colecovision\ \(GoodCol\ v2.01\)/ColecoVision\ BIOS\ \(1982\).col 
MD5 (/Users/tonypowell/Documents/Full_Rom_Sets/Colecovision (GoodCol v2.01)/ColecoVision BIOS (1982).col) = 2c66f5911e5b42b8ebe113403548eee7

Console info -

Jul 16 23:02:29 tony-powells-mac-pro-2 OpenEmu[1260]: Macally ICEKEY2 Keyboard
Jul 16 23:02:29 tony-powells-mac-pro-2 OpenEmu[1260]: OELibraryDatabase loadFromURL: 'file:///Users/tonypowell/Library/Application%20Support/OpenEmu/Game%20Library/'
Jul 16 23:02:29 tony-powells-mac-pro-2 OpenEmu[1260]: window did load
Jul 16 23:02:46 tony-powells-mac-pro-2 OpenEmuHelperApp[1265]: parent application is: OpenEmu
Jul 16 23:02:46 tony-powells-mac-pro-2 OpenEmuHelperApp[1265]: Macally ICEKEY2 Keyboard
Jul 16 23:02:46 tony-powells-mac-pro-2 OpenEmu[1260]: SETUP DONE.
Jul 16 23:02:46 tony-powells-mac-pro-2 OpenEmu[1260]: Display's nominal refresh rate is 59.95 Hz
Jul 16 23:02:47 tony-powells-mac-pro-2 OpenEmuHelperApp[1265]: Rate change 0.000000 -> 1.000000
 

2.  Select a game in Intellivision and I get a ROM did not load error

md5 hashtag -

MD5 (/Users/tonypowell/Downloads/Executive ROM, The (1978) (Mattel).int) = 62e761035cb657903761800f4437b8af
tony-powells-mac-pro-2:~ tonypowell$ md5 /Users/tonypowell/Downloads/GROM\,\ The\ \(1978\)\ \(General\ Instruments\)\ \[\!\].int 
MD5 (/Users/tonypowell/Downloads/GROM, The (1978) (General Instruments) [!].int) = 0cd5946c6473e42e8e4c2137785e427f
tony-powells-mac-pro-2:~ tonypowell$ md5 /Users/tonypowell/Downloads/IntelliVoice\ BIOS\ \(1981\)\ \(Mattel\).int 
MD5 (/Users/tonypowell/Downloads/IntelliVoice BIOS (1981) (Mattel).int) = d5530f74681ec6e0f282dab42e6b1c5f
tony-powells-mac-pro-2:~ tonypowell$ md5/Users/tonypowell/Downloads/Entertainment\ Computer\ System\ EXEC-BASIC\ \(1978\)\ \(Mattel\).7z 
-bash: md5/Users/tonypowell/Downloads/Entertainment Computer System EXEC-BASIC (1978) (Mattel).7z: No such file or directory
tony-powells-mac-pro-2:~ tonypowell$ md5 /Users/tonypowell/Documents/Full_Rom_Sets/Intellivision/Mattel\ Intellivision\ \(GoodINTV\ v2.03\)/Entertainment\ Computer\ System\ EXEC-BASIC\ \(1978\)\ \(Mattel\)\ \[\!\].int 
MD5 (/Users/tonypowell/Documents/Full_Rom_Sets/Intellivision/Mattel Intellivision (GoodINTV v2.03)/Entertainment Computer System EXEC-BASIC (1978) (Mattel) [!].int) = 2e72a9a2b897d330a35c8b07a6146c52

Console info -

Jul 16 23:15:19 tony-powells-mac-pro-2 OpenEmu[1326]: Macally ICEKEY2 Keyboard
Jul 16 23:15:19 tony-powells-mac-pro-2 OpenEmu[1326]: OELibraryDatabase loadFromURL: 'file:///Users/tonypowell/Library/Application%20Support/OpenEmu/Game%20Library/'
Jul 16 23:15:19 tony-powells-mac-pro-2 OpenEmu[1326]: window did load
Jul 16 23:15:33 tony-powells-mac-pro-2 OpenEmuHelperApp[1330]: parent application is: OpenEmu
Jul 16 23:15:33 tony-powells-mac-pro-2 OpenEmuHelperApp[1330]: Macally ICEKEY2 Keyboard
Jul 16 23:15:33 tony-powells-mac-pro-2 OpenEmuHelperApp[1330]: ROM did not load.

**Comment by CLIprogrammer** [comment]
El Capitan MAC OSX.  Downloaded several Atari 5200 (5200.ROM included) , Atari 7800 games and all programs successfully opened.  Having trouble with ColecoVision and Intellivision.

**Comment by clobber** [comment]
All systems work here. Specify an example game + md5 hash that does not load.

**Comment by CLIprogrammer** [comment]
Thank you for your comment.  I did as you asked for one game and discovered when I pulled both ColecoVision and Intellivision ROM sets that I didn't Unarchive ANY game as they were all suffixed .7z  I missed the suffix - too excited.  I Unarchived one game each for ColecoVision and Intellivision and they both worked.  Prior to your comment, I also searched through GitHub's closed tickets and found another Intellivision ticket with the same problem; but waited until this weekend for further research when I found your comment.  Again, many thanks!
